### PR TITLE
42degrees api clarifications in daemon.py

### PR DESCRIPTION
**Pull request by 42degrees**

### CODE REVIEW

File: lbrynet/daemon/Daemon.py [hunk 1]
```diff
@@ -523,7 +523,8 @@ def get_est_cost_from_uri(self, uri):
         defer.returnValue(result)
 
     def get_est_cost(self, uri, size=None):
-        """Get a cost estimate for a lbry stream, if size is not provided the
+        """
+		Get a cost estimate for a lbry stream, if size is not provided the
         sd blob will be downloaded to determine the stream size
 
         """
```
Comment (eukreign) [inline lbrynet/daemon/Daemon.py]: `Get a cost...` should be dedented to match how all of the other doc strings appear in `Daemon.py`. Also, please remove the blank line at the end.

File: lbrynet/daemon/Daemon.py [hunk 2]
```diff
@@ -2519,16 +2530,21 @@ def jsonrpc_claim_list_mine(self, account_id=None):
...
                     'category': (str) "claim", "update" , or "support"
+                    'channel_name': (str) name of the channel prefixed with '@'
                     'claim_id': (str) claim ID of the claim
                     'confirmations': (int) number of blocks of confirmations for the claim
+                    'decoded_claim': (bool) whether or not the claim value was decoded
                     'expiration_height': (int) the block height which the claim will expire
                     'expired': (bool) true if expired, false otherwise
+                    'has_signature': (bool) included if decoded_claim
                     'height': (int) height of the block containing the claim
+                    'is_pending': (bool) *** Don't know what this means.  I either get "false" or null for the value. ***,
```
Comment (eukreign) [inline lbrynet/daemon/Daemon.py]: This line is too long, it failed the linter during build.

File: lbrynet/daemon/Daemon.py [hunk 3]
```diff
@@ -2519,16 +2530,21 @@ def jsonrpc_claim_list_mine(self, account_id=None):
...
                     'is_spent': (bool) true if claim is abandoned, false otherwise
                     'name': (str) name of the claim
+                    'nout': (int) nout of the claim
                     'permanent_url': (str) permanent url of the claim,
+                    'signature_is_valid': (bool) indicates that the signature is valid, but is only included if has_signature is true,
```
Comment (eukreign) [inline lbrynet/daemon/Daemon.py]: This line is too long, it failed the linter during build.

File: lbrynet/daemon/Daemon.py (hunk 2 above)
Comment (jackrobison) [inline lbrynet/daemon/Daemon.py]: This value should be True if the claim has not yet been included in a block

Comment (tzarebczan) [reply lbrynet/daemon/Daemon.py]: @jackrobison see https://github.com/lbryio/lbry/issues/1470

**Comment by eukreign** [comment]
closing as it appears to be abandoned by PR author

**Pull request closed without merging**